airflow/www/static/js/connection_form.js

```diff
@@ -87,7 +40,7 @@ $(document).ready(function () {
       $(this).text($(this).attr("orig_text"));
     });
     $(".form-control").each(function(){$(this).attr('placeholder', '')});
-
+    let config = JSON.parse(decode($("#field_behaviours")[0].innerHTML))
```

Comment:
So at the least lets do:
```suggestion
    let config = JSON.parse(decode($("#field_behaviours").text()))
```

But I'm still not happy about needing that decode function.

:thinking:

Reply:
I see. Feel free :)

Reply:
K, lets go with this, and the suggestion I have made for a more robust escape methods

Reply:
Sure. I am just casual front-end developer :). Whatever is better :)